lib/Dialect/FIRRTL/Transforms/GrandCentral.cpp

```diff
@@ -1711,27 +1711,36 @@ void GrandCentralPass::runOnOperation() {
   instancePaths = &instancePathCache;
 
   /// Contains the set of modules which are instantiated by the DUT, but not a
-  /// companion or instantiated by a companion.  If no DUT exists, treat the top
-  /// module as if it were the DUT.  This works by doing a depth-first walk of
-  /// the instance graph, starting from the "effective" DUT and stopping the
-  /// search at any modules which are known companions.
+  /// companion, instantiated by a companion, or instantiated under a bind.  If
+  /// no DUT exists, treat the top module as if it were the DUT.  This works by
+  /// doing a depth-first walk of the instance graph, starting from the
+  /// "effective" DUT and stopping the search at any modules which are known
+  /// companions.
```

Comment:
```suggestion
  /// companions or marked to be lowered to bind.
```
?
(modulo phrasing, sorry)

Reply:
Fixed in: https://github.com/llvm/circt/pull/6741/commits/dbef859185bae85ee13b1c90e8c482d98e4a8224